src/weather-api.js

```diff
@@ -0,0 +1,16 @@
+async function getWeatherData(location, apiKey) {
```

Comment:
- [ ] It uses ES6 syntax. Kindly let's turn this function syntax to arrow function.
![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/es6_requirement.png)